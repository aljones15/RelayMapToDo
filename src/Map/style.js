import { flex, flatten } from '../../style';

const containerSize = flex(1, 1);
const container = {
  color: 'rgba(255,255,255,1.0)',
  backGroundColor: 'rgba(55, 55, 55, 1.0)',
};

export default flatten(containerSize, container);
