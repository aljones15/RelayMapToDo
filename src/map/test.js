import React from 'react';
import renderer from 'react-test-renderer';
import Map from './index';

describe('map should', () => {
  const mapJson = renderer.create(<Map />).toJSON();
  it('match a snapshot', () => {
    expect(mapJson).toMatchSnapshot();
  });
});
