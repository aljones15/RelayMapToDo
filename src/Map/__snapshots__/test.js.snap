// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`map should match a snapshot 1`] = `
<div
  id="map"
/>
`;
